src/main/java/org/jabref/gui/preferences/journals/AbbreviationViewModel.java

```diff
@@ -94,4 +94,18 @@ public boolean equals(Object o) {
     public int hashCode() {
         return Objects.hash(getName(), isPseudoAbbreviation());
     }
+
+    public boolean containsCaseIndependent(String s) {
```

Comment:
Mabye a better name for the variable: journalAbbrev?

Reply:
Does not necessarily have to be an abbreviation though, as we also search in the full name. What do you think about searchTerm?